packages/suite-desktop-core/src/libs/info.ts

```diff
@@ -1,11 +1,26 @@
 import { app } from 'electron';
-import si from 'systeminformation';
+import os from 'os';
 
 import { isDevEnv } from '@suite-common/suite-utils';
 import { bytesToHumanReadable } from '@trezor/utils';
 
 import { b2t } from './utils';
 
+const getCPUInfo = () => {
+    const cpus = os.cpus();
+    const cores = cpus.length;
+    const model = cpus[0]?.model.trim() ?? 'Unknown CPU';
+
+    const maxSpeedHz = Math.max(...cpus.map(cpu => cpu.speed));
```

Comment:
```suggestion
    const maxSpeedMHz = Math.max(...cpus.map(cpu => cpu.speed));
```

Then `/1000` to GHz makes sense

Reply:
Oh yes sorry, I can't count :see_no_evil: